src/app/space/create/codebases/workspaces-notification/workspaces-notification.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="workspaces-notification" [class.workspaces-notification-hide]="hide">
+  <div class="workspaces-notification-bar workspaces-notification-{{type}}">
+    <i class="pficon pficon-in-progress" *ngIf="type === 'info'"></i>
```

Comment:
Can you add the class `fa-spin` to make the progress icon spin in order to give the user some feedback?

Reply:
Will do!

Reply:
I didn't realize that spins. Nice!